Dockerfiles/centos8.Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV PYTHON python3
 ENV PIP pip3
 ENV PYTHONDONTWRITEBYTECODE 1
 
-ENV URL_GET_PIP "https://bootstrap.pypa.io/pip/get-pip.py"
+ENV URL_GET_PIP "https://bootstrap.pypa.io/pip/3.6/get-pip.py"
```

Comment:
centos:8 is still at the 8.4.2105 version, hence, using python 3.6.9

```bash
[root@46e02483c7ec /]# cat /etc/system-release
CentOS Linux release 8.4.2105
```

https://hub.docker.com/_/centos?tab=tags